operator/src/main/resources/scripts/startNodeManager.sh

```diff
@@ -255,12 +255,14 @@ EOF
     if [ "${ADMIN_PORT_SECURE}" = "true" ]; then
       admin_protocol="https"
     fi  
-    echo "AdminURL=$admin_protocol\\://${AS_SERVICE_NAME}\\:${ADMIN_PORT}" >> ${wl_props_file}
+    if [ "${ISTIO_ENABLED}" == "true" ]; then
+      echo "AdminURL=t3\\://${AS_SERVICE_NAME}\\:${ADMIN_PORT}" >> ${wl_props_file}
```

Comment:
Why is AdminURL forced to t3 here?

Reply:
If it needs to be t3 for some reason, then t3s should be used if the admin port is secured.

Reply:
How is ISTIO_ENABLED getting passed to the pod scripts (which in turn make use of AdminURL)?  I don't see it getting set by the Operator, and I don't see the sample's domain resource template specifying the value.   The former is preferable.  For an example of the Operator setting env vars for the introspector job and for wl pods, search for "LOG_HOME" in the Operator java source.

Reply:
In the first phase of Istio support, let's support just plain ports now. Will research the secure ports in the future as separate tasks.

ISTIO_ENABLED is passed via "kubernetes/samples/scripts/common/utility.sh"

Reply:
> ISTIO_ENABLED is passed via "kubernetes/samples/scripts/common/utility.sh"

Instead, it should be set by the operator for all jobs/pods when the operator detects the domain is istio-enabled.  Follow 'LOG_HOME' in the java source as an example...

Reply:
LOG_HOME is also provided in the create-domain-inputs.yaml:

# The in-pod location for domain log, server logs, server out, and node manager log files
# If not specified, the value is derived from the domainUID as /shared/logs/<domainUID>
logHome: /shared/logs/domain1

So I think we need include istio values in it too:

# experimental
# istioEnabled specifies whether or not it is deployed under Istio.
istioEnabled: false

# experimental
# istioReadinessPort specifies the WebLogic port for readiness probe which is used under Istio.
istioReadinessPort: 8888